consensus/enclave/api/src/lib.rs

```diff
@@ -202,9 +210,12 @@ pub trait ConsensusEnclave: ReportableEnclave {
     /// Retrieve the public identity of the enclave.
     fn get_identity(&self) -> Result<X25519Public>;
 
-    /// Retreive the block signing public key from the enclave.
+    /// Retrieve the block signing public key from the enclave.
     fn get_signer(&self) -> Result<Ed25519Public>;
 
+    /// Retrieve the fee public address from the enclave, as (SpendPublic, ViewPublic)
```

Comment:
Does this comment need updating? It looks like it might be referring to the old tuple